doc/ampi/manual.rst

```diff
@@ -348,13 +353,13 @@ adding this directory to the ``$PATH`` environment variable:
 
 These wrappers also allow the user to configure AMPI and Charm++-specific
 functionality.
-To use Isomalloc for transparently migrating user heap data, link with
-*-memory isomalloc*. To use a Charm++ load balancer, link a strategy or
-a suite of strategies in with *-module <LB>*. For example:
+For example, to automatically select a Charm++ load balancer at program
```

Comment:
We should mention, maybe in a footnote, that AMPI automatically links with CommonLBs

Reply:
Done